scripts/india_lgd/local_government_directory_districts/preprocess.py

```diff
@@ -0,0 +1,256 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import csv
+import difflib
+import pandas as pd
+
+manual_override = {
```

Comment:
nit: have a comment describing it? also, rename to MANUAL_OVERRIDE?

Reply:
👍🏼